### ISSUE TITLE
Make it possible to check for active profile in Shortcuts or even switch to a different profile

### ISSUE THREAD

**Issue by LiroyvH** [Artificial-Pancreas/iAPS#255]
@avouspierre pointed out (in #211) that Automations can be used to automagically trigger a temporary target when a work-out is started. That's very handy, but I came across a small problem when trying to set mine up;
This leads to the following feature request: 

Whilst I was trying to achieve the same, I came across the problem that, sometimes, (far) before I start a work-out: I enable a "sporting" profile with various settings (lower insulin administration, higher BG target, no SMB's, etc.). That's a bit of a problem, because if the automation is enabled: I'd have a profile active *and* the Work-Out automation would try to activate a temporary target. (I'm not sure if those stack, or if the temporary target overrides the profile. Didn't test.) 

It would be nice to be able to check whether or not a profile other than Normal is enabled, whether or not a temporary target is already enabled and it'd be even nicer if you can switch to another profile instead of activating a temporary target.

Therefore, it would be nice if:
1.) It's possible in shortcuts to check if anything other than the "normal" profile is enabled. (Ideally, it returns the name of the active profile (if it's something other than "normal"), so that you can filter with the IF condition. Eg: IF active profile == "sport" -> stop executing this Shortcut.)
2.) If it's possible to, instead of switching to a temporary target, switch to a different profile. (Of course it's important to either give a time limit to this profile OR add a secondary automation for when the work-out ends so that it reverts to normal profile. ;))

A caveat I see here might be that if someone changes the name of a Profile, that the Shortcut would start failing as it no longer can find the profile its trying to activate. Shortcuts would error out, so you'd quickly see that problem. However, maybe adding a unique identifier to the profile instead (eg: a number (and if one is deleted: others don't in- or decrement, they keep their value.)) would be safer/better. But also more work, so... Ya.

Thank you for your consideration. :)

**Comment by Jon-b-m** [comment]
Yes this can be done. I plan to add this soon. The foundation for this is already implemented.
